share/ruby-build/jruby-10.0.0.0

```diff
@@ -0,0 +1,2 @@
+require_java 8
```

Comment:
Since the release notes for JRuby 10 say that the minimum Java version was bumped to 21, shouldn’t this line also say `require_java 21` then?